to_sandbox.txt

```diff
@@ -1,10 +1,35 @@
 # New
+ofl/jersey10 # https://github.com/google/fonts/pull/7319
+ofl/jersey10charted # https://github.com/google/fonts/pull/7320
+ofl/jersey15 # https://github.com/google/fonts/pull/7335
+ofl/jersey15charted # https://github.com/google/fonts/pull/7336
+ofl/sedan/ # https://github.com/google/fonts/pull/7340
+ofl/sedansc/ # https://github.com/google/fonts/pull/7341
 
 # Upgrade
+ofl/dohyeon # https://github.com/google/fonts/pull/7207
+ofl/frankruhllibre # https://github.com/google/fonts/pull/7332
+ofl/jua # https://github.com/google/fonts/pull/7207
+ofl/kiranghaerang # https://github.com/google/fonts/pull/7207
+ofl/yeonsung # https://github.com/google/fonts/pull/7207
 
 # Designer profile
+catalog/designers/davidsargent # https://github.com/google/fonts/pull/7330
+catalog/designers/stephenhutchings # https://github.com/google/fonts/pull/7331
+catalog/designers/orangered # https://github.com/google/fonts/pull/7337
+catalog/designers/sebastiansalazar/ # https://github.com/google/fonts/pull/7339
 
 # Metadata / Description / License
+ofl/jacquard12 # https://github.com/google/fonts/pull/7302
+ofl/ojuju # https://github.com/google/fonts/pull/7333
+ofl/loveyalikeasister/ # https://github.com/google/fonts/pull/7346
+ofl/joan/ # https://github.com/google/fonts/pull/7351
+ofl/maitree/ # https://github.com/google/fonts/pull/7343
+ofl/mallanna/ # https://github.com/google/fonts/pull/7344
+ofl/mandali/ # https://github.com/google/fonts/pull/7344
+ofl/manjari/ # https://github.com/google/fonts/pull/7344
+ofl/martel/ # https://github.com/google/fonts/pull/7344 
+ofl/martelsans/ https://github.com/google/fonts/pull/7344
 
 # Sample texts
 # Deleted: lang/languages/sa_Sylo.textproto # https://github.com/google/fonts/pull/7248
```

Comment:
Let's add the latest categorization tagging data to the main branch csv file and the csv file path to the to_sandbox list before we merge the lists this week.

@emmamarichal @m4rc1e